xgi/drawing/xgi_pylab.py

```diff
@@ -14,6 +14,8 @@
 from matplotlib import cm
 from matplotlib.colors import LinearSegmentedColormap, ListedColormap
 
+from xgi.stats import EdgeStat, NodeStat
```

Comment:
Question: did you try a relative import and it didn't work, or did you not try a relative import? :)

Reply:
Fixed, thanks!